services/web-app/utils/use-query-state.js

```diff
@@ -6,47 +6,18 @@
  */
 
 import { useRouter } from 'next/router'
-import { useCallback, useEffect, useMemo } from 'react'
-
-export const queryTypes = {
-  string: {
-    parse: (v) => v,
-    serialize: (v) => `${v}`
-  },
-  integer: {
-    parse: (v) => parseInt(v),
-    serialize: (v) => Math.round(v).toFixed()
-  },
-  float: {
-    parse: (v) => parseFloat(v),
-    serialize: (v) => v.toString()
-  },
-  boolean: {
-    parse: (v) => v === 'true',
-    serialize: (v) => (v ? 'true' : 'false')
-  },
-  timestamp: {
-    parse: (v) => new Date(parseInt(v)),
-    serialize: (v) => v.valueOf().toString()
-  },
-  isoDateTime: {
-    parse: (v) => new Date(v),
-    serialize: (v) => v.toISOString()
-  },
-  object: {
-    parse: (v) => JSON.parse(v),
-    serialize: (v) => JSON.stringify(v)
-  }
+import queryString from 'query-string'
+import { useCallback, useEffect, useState } from 'react'
+
+const queryStringConfig = {
+  arrayFormat: 'bracket-separator',
+  arrayFormatSeparator: '|'
```

Comment:
Could you add a link in this to the query string docs that show how this array format works?

Reply:
Done!